parser-tsconfig.eslint.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Why we need this file, is it related to ts or eslint? (Also, this looks similar to tsconfig.json file, do we want to use the same?)

Reply:
I think the above reply https://github.com/oppia/oppia/pull/12558#discussion_r624875607 explains this as well.

Reply:
I've left a reply on the same thread!